features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/UploadCertificate.jsx

```diff
@@ -221,16 +223,17 @@ export default function UploadCertificate(props) {
                             onBlur={event => handleAliasOnChange(event.target.value)}
                             margin='normal'
                             variant='outlined'
-                            error={isAliasEmpty}
-                            helperText={isAliasEmpty ?
-                                <FormattedMessage
-                                    id='Apis.Details.Endpoints.GeneralConfiguration.UploadCertificate.alias.error'
-                                    defaultMessage='Alias should not be empty'
-                                /> :
-                                <FormattedMessage
-                                    id='Apis.Details.Endpoints.GeneralConfiguration.UploadCertificate.alias.helpertext'
-                                    defaultMessage='Alias for the certificate'
-                                />
+                            error={isAliasEmpty || aliasList.includes(alias)}
```

Comment:
Do we need this **aliasList.includes(alias)** here?

Reply:
Yes. If the Alias already exists it should display and error.

Reply:
got it.